base_tier_validation/models/tier_validation.py

```diff
@@ -626,11 +631,14 @@ def _notify_rejected_review_body(self):
         )
         if has_comment:
             comment = has_comment.mapped("comment")[0]
-            return _("A review was rejected by %(user)s. (%(comment)s)") % {
-                "user": self.env.user.name,
-                "comment": comment,
-            }
-        return _("A review was rejected by %s.") % (self.env.user.name)
+            return self.env._(
+                "A review was rejected by %(user)s. (%(comment)s)",
+                user=self.env.user.name,
+                comment=comment,
+            )
+        return self.env._(
+            "A review was rejected by %(user_name)s.", user_name=self.env.user.name
+        )
```

Comment:
Same as previous comments. It's a shame to invalidate existing translatable terms just to use a kword argument on a term with only 1 arg